src/types/DigitsConverter.d.ts

```diff
@@ -0,0 +1 @@
+declare type DigitsConverter<I = string, O = string> = (value: I) => O;
```

Comment:
Please prevent creating a Global typescript file(Developers should be able to import the `type` if they want) and your types file/folder should be alongside your functions.